src/config/process.ts

```diff
@@ -31,8 +31,6 @@ if (!!btcNetwork && !networks.includes(btcNetwork)) {
 export const BTC_NETWORK = btcNetwork as BtcNetwork
 
 export const tracingConfig = {
-  jaegerHost: process.env.JAEGER_HOST || "localhost",
```

Comment:
I'm sure we still have to inject the host/port of the agent - even if its no longer jaeger.

Reply:
running into some weird issue.

```
    TypeError: Cannot read properties of undefined (reading 'tracingServiceName')

      197 |   resource: Resource.default().merge(
      198 |     new Resource({
    > 199 |       [SemanticResourceAttributes.SERVICE_NAME]: tracingConfig.tracingServiceName,
          |                                                                ^
      200 |     }),
      201 |   ),
      202 | })

      at Object.tracingServiceName (src/services/tracing.ts:199:64)
      at Object.<anonymous> (src/domain/payments/payment-flow.ts:7:18)
      at Object.<anonymous> (src/domain/payments/index.ts:6:14)
      at Object.<anonymous> (src/domain/wallets/deposit-fee-calculator.ts:5:19)
      at Object.<anonymous> (src/domain/wallets/index.ts:6:14)
      at Object.<anonymous> (src/config/yaml.ts:14:18)
      at Object.<anonymous> (src/config/process.ts:5:15)
      at Object.<anonymous> (src/config/index.ts:6:31)
      at Object.<anonymous> (src/services/mongodb/index.ts:7:17)
      at Object.require (test/jest-e2e.setup.js:2:34)
```